CRM/Utils/Token.php

```diff
@@ -1243,7 +1243,7 @@ public static function getTokenDetails(
     $numberofContacts = count($contactIDs);
     $query = new CRM_Contact_BAO_Query($params, $returnProperties);
 
-    $details = $query->apiQuery($params, $returnProperties, NULL, NULL, 0, $numberofContacts);
+    $details = $query->apiQuery($params, $returnProperties, NULL, NULL, 0, $numberofContacts, TRUE, FALSE, TRUE, CRM_Contact_BAO_Query::MODE_CONTACTS, NULL, TRUE);
```

Comment:
With such a long list of parameters it suggests that this method should probably be refactored. For now though, I think splitting the parameter list into separate lines like in [this call](https://github.com/civicrm/civicrm-core/blob/master/api/v3/utils.php#L562) is more readable. I can't find anything specific about it in the coding standards so ignore this if you like.